sea-orm-cli/src/commands/migrate.rs

```diff
@@ -78,6 +78,7 @@ pub fn run_migrate_command(
     Ok(())
 }
 
+#[cfg(feature = "cli")]
 pub fn run_migrate_init(migration_dir: &str) -> Result<(), Box<dyn Error>> {
```

Comment:
@billy1624 is it appropriate to put `run_migrate_init` behind a feature guard?

Reply:
I feel like the main problem here is the fact that git2 depends on a binary library that, when shared has to be in the same version with might make some conflicts in projects involving sea-orm and some other crates that depend on git2. Plus sea-orm, as native in rust, can be compiled on any target. Such binary dependencies might break that as they did in that case. Can the code involving git2 library be replaced with non-dependency code? From what I see it only does:
```
    if git2::Repository::discover(Path::new(&migration_dir)).is_ok() {
        write_file!(".gitignore", "_gitignore");
    }
```

Reply:
Although if we can avoid CLI dependency with feature flags then I think it's just fine. In the end no one will use CLI on iOS.